### PR TITLE
Separating installation procedures for Debian and Ubuntu

### PR DESCRIPTION
**Pull request by panda1100**
Signed-off-by: Yoshiaki Senda yoshiaki@live.it

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -322,10 +322,30 @@ For the setuid installation do above command first and then this one:
...
+    $ wget https://github.com/{orgrepo}/releases/download/v{InstallationVersion}/{command}-suid_{InstallationVersion}_amd64.deb
+    $ sudo dpkg -i ./{command}-suid_{InstallationVersion}_amd64.deb
+
+Install Ubuntu packages
+------------------------------
+
+Pre-built Ubuntu packages are available on PPA (Personal Package Archive) for the amd64 and arm64 architectures.
 For the non-setuid installation use these commands:
 
 .. code::
```
Comment (DrDaveD) [inline installation.rst:351]: I tried the Ubuntu instructions in a fresh ubuntu:22.04 docker container and got the following:
```
# add-apt-repository -y ppa:apptainer/ppa
bash: add-apt-repository: command not found
```
I think the Ubuntu instructions need some more for bootstrapping on a fresh install.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
The new Debian install instructions look OK, but while you're at it fix up the Ubuntu instructions for starting from a minimal image.

File: installation.rst (hunk 1 above)
Comment (panda1100) [reply installation.rst:359]: Yes, That is true for container. Desktop/Sever version of Ubuntu has `add-apt-repository` command out of the box. 
I updated my PR.

**Comment by panda1100** [comment]
Sorry, I squashed commits and signed-off, Could you please help me to re-run DCO check?

File: installation.rst [hunk 2]
```diff
@@ -322,10 +322,39 @@ For the setuid installation do above command first and then this one:
...
+
+    $ wget https://github.com/{orgrepo}/releases/download/v{InstallationVersion}/{command}-suid_{InstallationVersion}_amd64.deb
+    $ sudo dpkg -i ./{command}-suid_{InstallationVersion}_amd64.deb
+
+Install Ubuntu packages
+------------------------------
+
+Pre-built Ubuntu packages are available on PPA (Personal Package Archive) for the amd64 and arm64 architectures. 
+
+First, on Ubuntu based container install `software-properties-common` package to obtain `add-apt-repository` command. On Ubuntu Desktop/Sever derived systems skip this step.
```
Comment (DrDaveD) [inline installation.rst:350]: Typo "Sever" -> "Server".  Also change "container" to "containers".

File: installation.rst [hunk 3]
```diff
@@ -322,10 +322,39 @@ For the setuid installation do above command first and then this one:
...
+
+.. code::
+
+    $ apt update
```
Comment (DrDaveD) [inline installation.rst:354]: This needs a sudo.  Also, some of the instructions on this page say "apt update" and some say "apt-get update".  Please choose one and make them consistent.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
I believe the problem with the DCO test is the format of the Sign-off.  It expects the email address to be surrounded by brackets `<>`.

I generally think it is a good idea to squash commits and force-push as you did, that's good.  If you don't do it I usually do a squash & merge.

File: installation.rst (hunk 2 above)
Comment (panda1100) [reply installation.rst:350]: Thank you!

File: installation.rst (hunk 3 above)
Comment (panda1100) [reply installation.rst:354]: I added `sudo` and made those consistent.

**Review by DrDaveD — APPROVED** [review]

**Comment by DrDaveD** [comment]
Great, now please submit the prior PR and this one to the 1.1 branch too.

**Comment by panda1100** [comment]
Will do! Thank you

**Pull request merged**